backend/rust-gbt/src/audit_transaction.rs

```diff
@@ -14,6 +14,7 @@ pub struct AuditTransaction {
     pub uid: u32,
     pub fee: u64,
     pub weight: u32,
+    pub vsize: u32,
```

Comment:
Perhaps this should be a different name...

Later on we do `(4 * next_tx.ancestor_vsize())` which to the untrained eye would look like an "oh, I'll just replace it with weight, it's the same thing... when it's not.

It's the MAX of vsize or sigops * 5.

`sigop_adjusted_vsize` maybe?

Reply:
yes, very good point